custom_components/tesla_custom/sensor.py

```diff
@@ -497,3 +507,50 @@ def native_value(self) -> Optional[datetime]:
         if self._car.charging_state in ["Charging", "Complete"]:
             return self._value
         return None
+
+class TeslaCarTpmsPressureSensor(TeslaCarEntity, SensorEntity):
+    """Representation of the Tesla car TPMS Pressure sensor."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        car: TeslaCar,
+        coordinator: TeslaDataUpdateCoordinator,
+        tpms_sensor: str,
+    ) -> None:
+        """Initialize TPMS Pressure sensor."""
+        super().__init__(hass, car, coordinator)
+        self._tpms_sensor = tpms_sensor
+        self.type = tpms_sensor
+        self._attr_device_class = SensorDeviceClass.PRESSURE
+        self._attr_state_class = SensorStateClass.MEASUREMENT
+        self._attr_native_unit_of_measurement = PRESSURE_BAR
+        self._attr_icon = "mdi:gauge-full"
+        # self._value: Optional[datetime] = None
+
+    @property
+    def native_value(self) -> float:
+        """Return TPMS Pressure."""
+        if self._tpms_sensor == "TPMS front left":
```

Comment:
This use of a string compare is brittle particularly when it's defined above as a constant. Why not replace the list with a dict with the key being the text string and the value being the attribute we want found?

That way:
```python
    return getattr(self._car, "TPMS_SENSORS.get(self._tpms_sensor))
```

Reply:
We can also add a default case where if there is no attribute, we return None or something.

Reply:
Very cool, I was looking for a way to do that! Thanks